components/ai_chat/resources/page/components/main/index.tsx

```diff
@@ -104,10 +107,57 @@ function Main() {
     }
   }
 
+  let inputFocused: boolean = false
+  let viewPortWithoutKeyboard = 0
+  let keyboardSize = 0
+
+  window.addEventListener('resize', () => {
+    if (!context.isMobile || !inputFocused || window.visualViewport == null) {
```

Comment:
Note: I don't think `visualViewport` will be set to null if it doesn't exist - more likely it'll be undefined.
```suggestion
    if (!context.isMobile || !inputFocused || !window.visualViewport) {
```